### PR TITLE
Drop 0.4 support and update old compat defs

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by musm** [comment]
is this g2g?

File: src/WinRPM.jl [hunk 1]
```diff
@@ -318,12 +317,12 @@ type RPMVersionNumber
     s::AbstractString
 end
 Base.convert(::Type{RPMVersionNumber}, s::AbstractString) = RPMVersionNumber(s)
-@compat Base.:(<)(a::RPMVersionNumber, b::RPMVersionNumber) = false
```
Comment (tkelman) [inline src/WinRPM.jl]: did you test this on 0.4?

Comment (musm) [inline src/WinRPM.jl]: oh right this still has 0.4 as the minimum, I guess this won't work

**Comment by musm** [comment]
Updated to drop 0.4

**Comment by musm** [comment]
Is this good now? Thanks

**Comment by tkelman** [comment]
This is the kind of package where it's likely most bug fixes would need to be backported, so I don't see a rush on this.

**Comment by musm** [comment]
Do u then think we can just tag master?

**Comment by musm** [comment]
Specifically for the 7z change

**Comment by musm** [comment]
Oh nvm I think you already tagged a release thanks!

**Comment by musm** [comment]
Closing since this is not a bugfix

**Pull request closed without merging**